MobileDevProject/ViewController.swift

```diff
@@ -25,10 +25,18 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
     var compassHeading: Double = 0
     var mapIsRotating: Bool = false
     var locationManager: CLLocationManager!
-    
-    var nearestBeaconCoordinate: CGPoint!
+    var nearestBeacon: BeaconInfo!
     var lineShapeLayer: CAShapeLayer!
+    var circleShapeLayer: CAShapeLayer!
+    var beaconsArray: [CLBeacon] = []
+    var circleShapeDrawn: Bool = false
+    var beacon1: CLBeacon!
+    var beacon2: CLBeacon!
+    var cgPoint: CGPoint!
```

Comment:
What does `cgPoint` mean? What does this variable do? And also remove the empty space.